styles.css

```diff
@@ -65,6 +65,65 @@ main {
   margin-right: 16px;
 }
 
+#mobile_menu {
+  height: 568px;
+  width: 100vw;
+  display: none;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+#mobile_menu_items {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+

```

Comment:
- [ ] **Good job on implementing the mobile menu 👍 👍** 
  however, it would be better to use a fixed position for the mobile menu items as using a ```fixed position``` for the mobile menu can improve the user experience, as it ensures that the menu remains accessible while scrolling through the page content. 😎😎
 
   ![image](https://user-images.githubusercontent.com/102757126/217014312-7d48c197-c575-4b1f-9e7c-3845bd4013ce.png)

   _It can also help provide a more consistent and predictable navigation experience. You may follow this[ link](https://developer.mozilla.org/en-US/docs/Web/CSS/position) for more info 👍_